css/style.css

```diff
@@ -15,5 +14,147 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+.topnav {
+  background-color: rgb(255, 255, 255);
+  overflow: hidden;
+  margin-right: 60px;
+}
+
+/* nav bar */
+.topnav a {
+  float: right;
+  color: #5a5a5a;
+  text-align: center;
+  padding: 1vw 2vw;
+  text-decoration: none;
+  font-size: 2vw;
+  background-color: white;
+  margin-top: 2vw;
+}
+
+/* the color of links on hover */
+.topnav a:hover {
+  color: orangered;
+}
+
+/* Adding bold font to the active/current link */
+.topnav a.active {
+  font-weight: bold;
+}
+.logo {
+  margin-left: 4vw;
+  margin-top: 2vw;
+  height: 3vw;
+  width: 2vw;
+}
+/* styling background image & texts on the image */
+
+.big-image {
+  width: 97%;
+  margin: 0;
+}
+
+.container {
+  position: relative;
+  text-align: center;
+}
+h1 {
+  font-family: "Roboto", sans-serif;
+  font-weight: 50;
+  font-size: 6vw;
+  font-stretch: condensed;
+}
+h5 {
+  font-family: "Roboto", sans-serif;
+  font-weight: 50;
+  font-size: 2.5vw;
+  font-stretch: condensed;
+  width: 100%;
+}
+.text-container {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: white;
+  text-align: center;
+}
+.button1 {
+  background-color: orangered;
+  border: 2vw solid orangered;
+  border-radius: 3%;
+  width: 30%;
+  font-size: 1.5vw;
+}
+
+.article-summary {
```

Comment:
this is one way to do this but after what we learned in class last week, try to have a go at using flex-box instead to achieve the same effect